Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11.6-alpine3.9
+FROM golang:1.12.5-alpine3.9
```

Comment:
We're using go 1.12.x in the Travis VM, so might as well update our Docker environment to use it too.